src/main/flight/pid.c

```diff
@@ -181,10 +181,10 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .abs_control_error_limit = 20,
         .abs_control_cutoff = 11,
         .antiGravityMode = ANTI_GRAVITY_SMOOTH,
-        .dterm_lowpass_hz = 100,    // dual PT1 filtering ON by default
-        .dterm_lowpass2_hz = 200,   // second Dterm LPF ON by default
-        .dterm_filter_type = FILTER_PT1,
-        .dterm_filter2_type = FILTER_PT1,
+        .dterm_lowpass_hz = 150,    // dual BIQUAD filtering ON by default
```

Comment:
I'd probably add a comment here (and for `gyro_lowpass_hz`) stating that this is set only for the benefit of Betaflight configurator 10.4, but the filter is actually defaulting to 'off', to prevent future attempts at 'fixing' this.